roles/pulp3-devel/tasks/main.yml

```diff
@@ -0,0 +1,108 @@
+---
+- block:
+
+  - name: Install several useful packages
+    package:
+      name:
+        - bash-completion
+        - dnf-utils
+        - dstat
+        - fd-find
+        - fedora-easy-karma
+        - fpaste
+        - fzf
+        - git
+        - htop
+        - httpie
+        - iotop
+        - jnettop
+        - jq
+        - ncdu
+        - python-django-bash-completion
+        - python-ipdb
+        - python-rpdb
```

Comment:
same as ipdb

Reply:
on second thought, i think take this out. many on the team use pycharm's remote debugger, and i almost exclusively use ipdb while running processes in the foreground. if this annoys people, they can always add it back.

if this goes, telnet should also go.